docs/rules/terraform_module_pinned_source_is_semver.md

```diff
@@ -0,0 +1,54 @@
+# terraform_module_pinned_source_is_semver
```

Comment:
The naming feels a little redundant. How about `terraform_module_semver_source`?